.github/workflows/sync-providers.yaml

```diff
@@ -9,7 +9,12 @@ on:
     secrets:
       personal_access_token:
         required: true
+      JIRA_API_TOKEN:
+        required: true
 
+env:
```

Comment:
are those env variables  will be visible under the repo settings/env ? better to define them there